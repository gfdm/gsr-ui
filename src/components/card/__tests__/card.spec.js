import { mount, createLocalVue } from '@vue/test-utils'
import IdCard from '../'

const localVue = createLocalVue()
localVue.use(IdCard)

describe('IdCard', () => {
  test('should render well', () => {
    const Wrapper = {
      name: 'wrap',
      template: `
        <id-card></id-card>
      `
    }

    const wrapper = mount(Wrapper, {
      localVue
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
