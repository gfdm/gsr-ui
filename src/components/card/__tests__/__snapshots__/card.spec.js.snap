// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdCard should render well 1`] = `
<section class="id-card">
  <header class="id-card__header"><span>Default</span></header>
  <section class="id-card__body">
    Content
  </section>
</section>
`;
